src/app/core/services/page-title/page-title.service.ts

```diff
@@ -0,0 +1,127 @@
+import {Injectable} from '@angular/core';
+import {Title} from '@angular/platform-browser';
+import {Subject, of} from 'rxjs';
+import {takeUntil, switchMap, tap} from 'rxjs/operators';
+
+import {ProjectService, ClusterService, ApiService} from '../../services';
+import {ParamsService, PathParam} from '../../services/params/params.service';
+
+export enum PageView {
```

Comment:
This is basically a copy of `View` enum from `auth.guard`. Export that to a common place i.e. `shared/entity`.